sphinx/pycode/ast.py

```diff
@@ -202,7 +202,10 @@ def is_simple_tuple(value: ast.AST) -> bool:
             return "%s[%s]" % (self.visit(node.value), self.visit(node.slice))
 
     def visit_UnaryOp(self, node: ast.UnaryOp) -> str:
-        return "%s %s" % (self.visit(node.op), self.visit(node.operand))
+        # UnaryOp is one of {UAdd, USub, Invert, Not}. Only Not needs a space.
```

Comment:
I think adding examples here makes the comment clearer:
```suggestion
        # UnaryOp is one of {UAdd, USub, Invert, Not}, which correspond 
        # respectively to `+x`, `-x`, `~x`, and `not x`. Only Not needs a space.
```